src/main/java/ua/ukr/net/Dao/JdbcDepartmentDao.java

```diff
@@ -0,0 +1,33 @@
+package ua.ukr.net.Dao;
```

Comment:
Rename please package Dao to dao. Package name should start with lower case letter